test/versioned/disabled-instrumentation/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "disabled-instrumentation-tests",
+  "targets": [],
+  "version": "0.0.0",
+  "private": true,
+  "tests": [
+    {
+      "engines": {
+        "node": ">=18"
+      },
+      "dependencies": {
+        "express": "latest",
```

Comment:
We should at least pin this one to v4 until we can figure out what is happening with v5 and the security agent.